features/bootstrap/ServiceTemplateConfigurationContext.php

```diff
@@ -388,11 +388,21 @@ function ($context) {
      */
     public function iDeleteAServiceTemplate()
     {
-        $this->currentPage = new ServiceTemplateConfigurationListingPage($this);
-        $object = $this->currentPage->getEntry($this->initialProperties['description']);
-        $this->assertFind('css', 'input[type="checkbox"][name="select[' . $object['id'] . ']"]')->check();
-        $this->setConfirmBox(true);
-        $this->selectInList('select[name="o1"]', 'Delete');
+        try {
+            $this->spin(
```

Comment:
I do not understand why spin() is even needed. Deletion is supposed to be synchronous, therefore a spin() on the ServiceTemplateConfigurationListingPage should not be needed. Am I right ?